EasyPost/Models/API/WebhookCustomHeader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EasyPost.Models.API
+{
+    /// <summary>
+    ///     Class for webhook custom header
+    /// </summary>
+    public class WebhookCustomHeader
+    {
+        #region JSON Properties
+
+        /// <summary>
+        ///     Custom Header name
+        /// </summary>
+        [JsonProperty("name")]
+        public string? Name { get; set; }
```

Comment:
```suggestion
        public string Name { get; set; }
```

issue: I don't believe these are optional, they're either there or not at all